core/Cargo.toml

```diff
@@ -137,7 +137,7 @@ sha2 = "0.10.8"
 snap = "1.1.0"
 speedb = { version = "0.0.4", features = ["lz4", "snappy"], optional = true }
 storekey = "0.5.0"
-surrealml-core = { version = "0.0.7", optional = true }
+surrealml-core = { version = "0.0.8", optional = true }
```

Comment:
v0.0.8 is not backwards compatible, so we can't update it like this. In my [dependency update PR](https://github.com/surrealdb/surrealdb/pull/3385) I [added an ml2 feature flag](https://github.com/surrealdb/surrealdb/pull/3385/commits/edcc7d145e19a13756669ce318b5059e64faf738).